pkg/installation/install_test.go

```diff
@@ -151,6 +153,14 @@ func Test_removeLink_notExists(t *testing.T) {
 	}
 }
 
+func Test_removeKrew(t *testing.T) {
+	envPath := environment.MustGetKrewPaths()
+	expectedErrorMessage := fmt.Sprintf("removing krew is not allowed through krew. Please run:\n rm -r %s", envPath.BasePath())
+	if err := Remove(envPath, "krew"); err.Error() != expectedErrorMessage {
```

Comment:
I'd probably do a `Contains` check with "not allowed" string here. It's tad bit too strong.